src/console/App.test.js

```diff
@@ -1,8 +1,9 @@
 import App from './App';
+import { wrapMockStore } from '../normandy/tests/mockStore';
```

Comment:
nit: we should use absolute paths using the `aliases` defined in `config-overrides.js`. Instead of using the relative path to get to `../normandy/...`, we have an alias so we can just use `normandy` and webpack knows where to look.

This is more useful if you're in a deeply-nested file, where `'../../../normandy/...'` is replaced by `'normandy/...'`, but we should remain consistent.